crud/pom.xml

```diff
@@ -90,6 +90,11 @@
             <groupId>org.slf4j</groupId>
             <artifactId>slf4j-simple</artifactId>
         </dependency>
+        <dependency>
+            <groupId>com.redhat.lightblue.hook</groupId>
+            <artifactId>lightblue-audit-hook</artifactId>
+            <scope>test</scope>
+        </dependency>
```

Comment:
If that's  lightblue-rest/crud/pom.xml, it shouldn't.

On Mon, Apr 20, 2015 at 8:07 AM, Dennis Crissman <notifications@github.com>
wrote:

> In crud/pom.xml
> <https://github.com/lightblue-platform/lightblue-rest/pull/134#discussion_r28691259>
> :
>
> > @@ -90,6 +90,11 @@
> >              <groupId>org.slf4j</groupId>
> >              <artifactId>slf4j-simple</artifactId>
> >          </dependency>
> > +        <dependency>
> > +            <groupId>com.redhat.lightblue.hook</groupId>
> > +            <artifactId>lightblue-audit-hook</artifactId>
> > +            <scope>test</scope>
> > +        </dependency>
>
> Doesn't this create a circular dependency between core and audit-hook?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/lightblue-platform/lightblue-rest/pull/134/files#r28691259>
> .
>